MAINTAINERS.md

```diff
@@ -6,5 +22,91 @@
 | Peter Nied | [peternied](https://github.com/peternied) | Amazon |
 | Barani Bikshandi | [bbarani](https://github.com/bbarani) | Amazon |
 | Sayali Gaikawad | [gaiksaya](https://github.com/gaiksaya) | Amazon |
+| Abhinav Gupta | [abhinavGupta16](https://github.com/abhinavGupta16) | Amazon |
+| Zelin Hao | [zelinh](https://github.com/zelinh) | Amazon |
+| Elaine | [zylzulu](https://github.com/zylzulu) | Amazon |
+
+[This document](https://github.com/opensearch-project/.github/blob/main/MAINTAINERS.md) explains what maintainers do in this repo, and how they should be doing it. If you're interested in contributing, see [CONTRIBUTING](CONTRIBUTING.md).
+
+## Zenhub Process Workflow
+We follow agile methodologies for our development and release process. We use zenhub to manage and track our stories and issues to effectively manage them over the sprint.
+
+### What is [Zenhub](https://www.zenhub.com/)?
+
+ZenHub is an agile project management and product roadmaps solution, natively integrated into GitHub. It is free to use for opensource repository and comes with a paid membership to manage private repositories. 
+We currently use zenhub only with our public and opensource repositories.
+
+### Setting up Zenhub
+
+Zenhub can be easily added as an [extension]((https://www.zenhub.com/extension)) to chrome and firefox which can be downloaded for free from the zenhub website. 
+Alternatively, we can use a the zenhub [webapp link](https://app.zenhub.com/workspaces/engineering-effectiveness-614cf4272a385f0015d2b48f/board?repos=357723952,406037663) to view the board.
+
+### Sprint Board aka Zenhub board aka Kanban board
+Once the zenhub extension is installed, zenhub board can be accessed using the Zenhub tab on github.
```

Comment:
I like pictures for describing user experiences, here is one that could work well here
![image](https://user-images.githubusercontent.com/2754967/143924779-c53177f8-922b-41a6-9c9a-54c899f3aae4.png)